index.js

```diff
@@ -197,6 +197,21 @@ async function checkHeartbeat() {
   }
 }
 
+function reformatHeartbeatStates(stateTransitionsArrays) {
+  const reasonsTable = ['movement', 'no_movement', 'door_open', 'initial_timer', 'duration_alert', 'stillness_alert']
+  const stateTransitionsObjects = []
+  function objToArray(value) {
+    const stateObject = {
+      state: value[0],
+      reason: reasonsTable[value[1]],
+      time: value[2],
+    }
+    stateTransitionsObjects.push(stateObject)
+  }
+  stateTransitionsArrays.forEach(objToArray)
```

Comment:
This feels a bit awkward. It is not generally good practice for a function to modify a variable outside of it like this.
What would feel more natural to me is if you your `objToArray` function simply returned `stateObject`.

Then, instead of `stateTransitionsArrays.forEach(objToArray)`, you can use 
`const stateTransitionObjects = stateTransitionsArrays.map(objToArray)`
See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map for more details about `map`.

You could also move the `objToArray` function (after giving it a more descriptive name) outside the `reformatHeartbeatStates` so that it doesn't need to be redefined every time `reformatHeartbeatStates` is called.